src/avro_ocf.erl

```diff
@@ -253,10 +255,23 @@ decode_block(_Lkup, _Type, _Codec, <<>>, 0, Acc, _Options) -> Acc;
 decode_block(Lkup, Type, deflate, Bin, Count, Acc, Options) ->
   Decompressed = zlib:unzip(Bin),
   decode_block(Lkup, Type, null, Decompressed, Count, Acc, Options);
+decode_block(Lkup, Type, snappy, Bin, Count, Acc, Options) ->
+  Size = byte_size(Bin),
+  <<Compressed:(Size-4)/binary, Checksum:4/binary>> = Bin,
+  {ok, Decompressed} = snappyer:decompress(Compressed),
+  case is_invalid_checksum(Decompressed, Checksum) of
+    false -> Decompressed;
```

Comment:
I'm not a big fan of double-negatives. Can you change the predicate to the positive `is_valid_checksum/2` and put the `true` case first?

Reply:
Changed! I have renamed the function to `is_valid_checksum`